src/test/regress/expected/rollback_to_savepoint.out

```diff
@@ -1,27 +0,0 @@
--- Regression test for this issue:
--- https://github.com/citusdata/citus/issues/3622
-SET citus.shard_count = 4;
-SET citus.next_shard_id TO 1954000;
-CREATE SCHEMA rollback_to_savepoint;
-SET search_path TO rollback_to_savepoint;
-CREATE TABLE t(a int);
-SELECT create_distributed_table('t', 'a');
- create_distributed_table
----------------------------------------------------------------------
-
-(1 row)
-
--- This timeout is chosen such that the INSERT with
--- generate_series(1, 100000000) is cancelled at the right time to trigger the
--- bug
-SET statement_timeout = '2s';
-BEGIN;
-INSERT INTO t VALUES (4);
-SAVEPOINT s1;
```

Comment:
cn we maybe do SET client_min_messages TO ERROR if the problem is warnings?

Reply:
The regressions.diff doesn't just contain warnings, the final line is an actual error:
```
 INSERT INTO t SELECT i FROM generate_series(1, 100) i;
+WARNING:  connection to the remote node localhost:57638 failed with the following error: 
+WARNING:  
+CONTEXT:  while executing command on localhost:57638
+ERROR:  connection to the remote node localhost:57638 failed with the following error: 
 ROLLBACK;
```